pyspeckit/spectrum/plotters.py

```diff
@@ -320,11 +343,12 @@ def reset_limits(self,xmin=None, xmax=None, ymin=None, ymax=None,
                                   ypeakscale=ypeakscale, **kwargs)
                 return
             
-            if (self.Spectrum.data.max() < self.ymin or self.Spectrum.data.min() > self.ymax
-                    or reset_ylimits):
-                if not self.silent and not reset_ylimits: warn( "Resetting Y-axis min/max because the plot is out of bounds." )
-                self.ymin = None
-                self.ymax = None
+            if self.ymin and self.ymax:
```

Comment:
I made this change in one of the first commits but I remember it had to do with a NoneType error, I'll run the examples again without it to make sure.